src/extension/kernel.ts

```diff
@@ -55,8 +56,10 @@ export class Kernel implements Disposable {
 
   protected cellManager = new NotebookCellManager(this.#controller)
 
+  protected serializer?: SerializerBase
```

Comment:
Where do we use this property in the Kernel?

Reply:
Good catch, this is an artifact of an old implementation.